### ISSUE TITLE
Odd TS highlight/hover when hovering a comment

### ISSUE THREAD

**Issue by Tyriar** [microsoft/TypeScript#56329]
When hovering "similar" in the comment here:

https://github.com/microsoft/vscode/blob/dc78919c39904ae6ef624d62e63c59a62a2b6b0d/src/vs/workbench/contrib/terminal/browser/terminalRunRecentQuickPick.ts#L76-L83

It thinks I'm highlighting `replace`:

![image](https://github.com/microsoft/vscode/assets/2193314/39959fad-11dd-4c69-99a7-3edc817b2da6)

**Comment by RyanCavanaugh** [comment]
I think the implied request here is that if a hover request lands inside a comment, even if that comment is "part of" a call node due to being inside the trivia, just don't show anything? @Tyriar ?

**Comment by Tyriar** [comment]
@RyanCavanaugh yep that would do it. What was extra strange about the comment hover is that it's highlighting several symbols, not just the `replace`
